recipes/base_install.rb

```diff
@@ -237,13 +237,14 @@
 include_recipe "aws-parallelcluster::_lustre_install"
 
 # Install EFA
-if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon'
+if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon' || (node['platform'] == 'ubuntu' && node['platform_version'] == "16.04")
   unless node['cfncluster']['cfn_region'].start_with?("cn-")
     include_recipe "aws-parallelcluster::_efa_install"
-  else
-    package 'openmpi-devel' do
-      retries 3
-      retry_delay 5
-    end
+  end
```

Comment:
this should be an "else":
if (not cn-) then execute efa_install
else execute mpi install

Reply:
Fixed